src/Test/BlockServiceTestCase.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\BlockBundle\Test;
+
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Sonata\BlockBundle\Block\BlockContextInterface;
+use Sonata\BlockBundle\Block\BlockContextManager;
+use Sonata\BlockBundle\Block\BlockContextManagerInterface;
+use Sonata\BlockBundle\Block\BlockLoaderInterface;
+use Sonata\BlockBundle\Block\BlockServiceInterface;
+use Sonata\BlockBundle\Block\BlockServiceManagerInterface;
+use Sonata\BlockBundle\Model\BlockInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Abstract test class for block service tests.
+ *
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+abstract class BlockServiceTestCase extends TestCase
+{
+    /**
+     * @var MockObject|ContainerInterface
+     */
+    protected $container;
+
+    /**
+     * @var MockObject|BlockServiceManagerInterface
+     */
+    protected $blockServiceManager;
+
+    /**
+     * @var BlockContextManagerInterface
+     */
+    protected $blockContextManager;
+
+    /**
+     * @var FakeTemplating
+     */
+    protected $templating;
+
+    protected function setUp()
```

Comment:
```suggestion
    protected function setUp(): void
```

Reply:
We can't do this. This will not work in PHP 7.1 :(

Reply:
I don't understand why this will not work. Could you please explain?

Reply:
`void` needs PHP 7.1
https://wiki.php.net/rfc/void_return_type

Reply:
Sure, so "This will not work in PHP < 7.1"
It's now clarified to me :+1:

Reply:
Currently, we're requiring `7.1` as minimum version:
https://github.com/sonata-project/SonataBlockBundle/blob/d0a67fcd6873e234467cd58b35aa3feb5b54a6ff/composer.json#L23

Reply:
Oh sorry, I was adressing the wrong problem. The problem is PHP 7.1 itself:
```
Fatal error: Declaration of Sonata\BlockBundle\Tests\Block\Service\MenuBlockServiceTest::setUp() must be compatible with Sonata\BlockBundle\Test\BlockServiceTestCase::setUp(): void in /app/tests/Block/Service/MenuBlockServiceTest.php on line 26
```
You can't override the method signature in PHP 7.1. The PHPUnit method signature was changed:
https://github.com/sebastianbergmann/phpunit/blob/7.5/src/Framework/TestCase.php#L407
https://github.com/sebastianbergmann/phpunit/blob/8.2/src/Framework/TestCase.php#L411